lib/travis/build/script/elm.rb

```diff
@@ -3,13 +3,13 @@ module Build
     class Script
       class Elm < NodeJs
         # Default NodeJS version to install
-        DEFAULT_NODE_VERSION = '10.13.0'
+        DEFAULT_NODE_VERSION = '13.11.0'
```

Comment:
imo, we should stick to [node LTS (even-numbered) releases](https://nodejs.org/en/about/releases/), which would mean '12.16.1' for now

Reply:
```suggestion
        DEFAULT_NODE_VERSION = '12.16.1'
```

I agree, LTS is better here. Would specifying "12"?

Reply:
This value should be passed to `nvm`, so if `nvm` understands it (I think it does), it's fine.

Reply:
```suggestion
        DEFAULT_NODE_VERSION = '12'
```

I agree, LTS is better here. Would specifying "12" work?